src/main/java/jdbchomework/utils/ConnectionUtil.java

```diff
@@ -0,0 +1,39 @@
+package jdbchomework.utils;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+
+    private static Connection connection;
+
+    public static Connection getConnection() {
+        FileInputStream fis;
+        Properties property = new Properties();
+        try {
+            fis = new FileInputStream("src/main/resources/config.properties");
+            property.load(fis);
+            String url = property.getProperty("db.url");
+            String user = property.getProperty("db.user");
+            String password = property.getProperty("db.password");
+            connection = DriverManager.getConnection(url, user, password);
+        } catch (IOException e) {
+            throw new RuntimeException("Cannot find file ", e);
+        } catch (SQLException e) {
+            throw new RuntimeException("Cannot get connection to db", e);
+        }
+        return connection;
+    }
+
+    public static void closeConnection() {
+        try {
+            connection.close();
+        } catch (SQLException e) {
+            throw new RuntimeException("Cannot close connection", e);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS00112)